napari/layers/shapes/shape_models/rectangle.py

```diff
@@ -1,17 +1,17 @@
 import numpy as np
 from xml.etree.ElementTree import Element
 from .shape import Shape
-from ..shape_util import find_corners, rectangle_to_box, poly_to_mask
+from ..shape_util import find_corners, rectangle_to_box
 
 
 class Rectangle(Shape):
     """Class for a single rectangle
 
     Parameters
     ----------
-    data : (4, 2) or (2, 2) array
-        Either a (2, 2) array specifying the two corners of an axis aligned
-        rectangle, or a (4, 2) array specifying the four corners of a bounding
+    data : (4, D) or (2, D) array
+        Either a (2, D) array specifying the two corners of an axis aligned
+        rectangle, or a (4, D) array specifying the four corners of a bounding
```

Comment:
Is an nD axis aligned rectangle uniquely defined by two points? I think actually not?

Reply:
you are correct it is weird, and the code actually only supports the two point version for 2D shapes, I have updated the doc string to reflect this